app/services/partners/family_request_create_service.rb

```diff
@@ -19,6 +20,7 @@ def call
       return self unless valid?
 
       request_create_svc = Partners::RequestCreateService.new(
+        request_type: @request_type.to_i,
```

Comment:
This service should already know the request type just by whether `for_families` is true or false.

Reply:
I feel that defining request_type in this way is more transparent.  Would you prefer to add logic in the controller to determine request_type rather than using the instance var?

Reply:
I'd prefer putting the business logic as far back as possible. The controller is calling a service, which already has to know what kind of request it's creating. We can rely on the service to set the request type.